.ci-config/getNewAmendments.js

```diff
@@ -0,0 +1,56 @@
+const xrpl = require("xrpl");
+const fs = require("fs");
+const path = require("path");
+
+const filePath = path.resolve(__dirname, "./rippled.cfg");
+const existingConfig = fs.readFileSync(filePath, "utf-8");
+
+const networkToEmulate = "wss://s.devnet.rippletest.net:51233/";
+
+const amendmentsToIgnore = [
+  "86E83A7D2ECE3AD5FA87AB2195AE015C950469ABF0B72EAACED318F74886AE90", // CryptoConditionsSuite is obsolete
+];
+
+async function main() {
+  const client = new xrpl.Client(networkToEmulate);
+  await client.connect();
+
+  // Looks up what amendments have been enabled via their hash
+  const request = {
+    command: "ledger_entry",
+    index: "7DB0788C020F02780A673DC74757F23823FA3014C1866E72CC4CD8B226CD6EF4",
```

Comment:
Nit: you can import the constant from `xrpl`.

Reply:
I don't think it's properly exported outside the repo. Importing it directly would require compiling the typescript, which would make it more annoying to run the script, so I don't think it's worth doing that for this.

Reply:
Will need to look into how why it is not on the exported object.